src/fu-device-metadata.h

```diff
@@ -64,4 +64,23 @@ G_BEGIN_DECLS
  */
 #define FU_DEVICE_METADATA_UEFI_CAPSULE_FLAGS	"UefiCapsuleFlags"
 
+/**
+ * FU_DEVICE_METADATA_FLASHROM_DEVICE_KIND:
+ *
+ * The type of flashrom device, e.g. "system-firmware" or "device-firmware"
```

Comment:
You mention device-firmware here, but it's actually not used anywhere else that I can tell.